src/components/Clock.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Clock.scss';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  currentTime: string;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    currentTime: new Date().toLocaleTimeString(),
+  };
+
+  timerId: NodeJS.Timer = setInterval(() => {}, 0);
```

Comment:
Для чого створювати інтервал який ти нігде не прибираєш і він просто грузить систему?)
```suggestion
  timerId?: NodeJS.Timer;
```